flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -33,6 +37,18 @@ class FlxGamepad implements IFlxDestroyable
 	public var model(default, set):FlxGamepadModel;
 	
 	/**
+	 * For gamepads that can have things plugged into them (the Wii Remote, basically).
+	 * Making the user set this helps Flixel properly interpret inputs properly. 
+	 * EX: if you plug a nunchuk into the Wii Remote, you will get different values for 
+	 * certain buttons than with the Wii Remote alone.
+	 * (This is probably why Wii games ask the player what control scheme they are using.)
+	 * 
+	 * In the future, this could also be used for any attachment that exposes new API features
+	 * to the controller, e.g. a microphone or headset
+	 */
+	public var attachment(default, set):FlxGamepadModelAttachment;
```

Comment:
In theory, yes; but let's wait until we actually have a use case for that ... [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) and all that.

Unless you can think of something super obvious that we could test with.